instrumentation/camunda/camunda-7.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/camunda/v7_0/behavior/CamundaCommonBehaviorModule.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.camunda.v7_0.behavior;
+
+import static io.opentelemetry.javaagent.extension.matcher.AgentElementMatchers.hasClassesNamed;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.extension.instrumentation.InstrumentationModule;
+import io.opentelemetry.javaagent.extension.instrumentation.TypeInstrumentation;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import net.bytebuddy.matcher.ElementMatcher;
+
+@AutoService(InstrumentationModule.class)
+public class CamundaCommonBehaviorModule extends InstrumentationModule {
+
+  public CamundaCommonBehaviorModule() {
+    super("camunda", "camunda-7.0", "camunda-behavior", "camunda-behavior-7_18");
+  }
+
+  @Override
+  public List<TypeInstrumentation> typeInstrumentations() {
+    return Collections.singletonList(new CamundaCommonBehaviorInstrumentation());
+  }
+
+  @Override
+  public ElementMatcher.Junction<ClassLoader> classLoaderMatcher() {
+    return hasClassesNamed(
+        "org.camunda.bpm.engine.impl.bpmn.behavior.TaskActivityBehavior",
+        "org.camunda.bpm.engine.impl.bpmn.behavior.NoneEndEventActivityBehavior",
+        "org.camunda.bpm.engine.impl.bpmn.behavior.ErrorEndEventActivityBehavior");
+  }
```

Comment:
Usually you don't need this. Matchers apply as classes are loaded, no need to check whether the classes you intend to instrument are present. Typically this method is used when instrumentation needs some class that is present only starting from some version.